tests/route/test_static_route.py

```diff
@@ -161,7 +161,18 @@ def _check_routes():
                 return False
         return True
 
-    assert (wait_until(60, 15, 0, _check_routes))
+    assert (wait_until(60, 15, 0, _check_routes)), (
+        "Failed to verify route redistribution. "
+        "The expected prefix '{}' was not found in the advertised routes of all BGP neighbors. "
```

Comment:
you have {} in the string, but no variable at the end of it. This's a syntax error. Please avoid those syntax error in script, it will be a bad regression to case.

Don't need to add those indications in failed reason, that's not related to the failed reason